library/src/main/java/com/feedhenry/sdk/sync/android/AndroidFileStorage.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright Red Hat, Inc, and individual contributors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.feedhenry.sdk.sync.android;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import com.feedhenry.sdk.sync.IFileStorage;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.lang.ref.WeakReference;
+
+/**
+ * Storage in Android local data store
+ */
+
+public class AndroidFileStorage implements IFileStorage {
+
+    private final WeakReference<Context> mContextWeak;
+
+    public AndroidFileStorage(Context ctx) {
+        mContextWeak = new WeakReference<>(ctx);
+    }
+
+    @Override
+    public FileInputStream openFileInput(@NonNull String pFilePath) throws FileNotFoundException {
```

Comment:
-1 to any prefixes on the variable name.  Ie filePath not pFilePath.

Reply:
ok, I was little bit surprised with this too, but it's already there. 
Do we have any guidelines for this? btw. what about Google recommendation for mSomething private and sSomething static?

Reply:
@wojta Google recommendation is idiot :) The recommendation is use whatever human readable. I mean `pFilePath` does not makes sense. `filePath` do :wink:

Reply:
Google says not to do it 

>  In Google Style special prefixes or suffixes, like those seen in the examples name_, mName, s_name and kName, are not used.

They are great for example code, but shouldn't be in the actual APIs.

https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names

Reply:
@wojta It was already in the rest of the SDK because the SDK was ported from Javascript where you need something like this because the language is awful.

Reply:
ok, I'll create a small task in JIRA to syncing the code style in separated SDK.